tmc-langs/src/lib.rs

```diff
@@ -524,6 +524,21 @@ pub fn login_with_password(
     Ok(token)
 }
 
+/// Authenticates with the server, returning a login Token.
+/// Password as parameter.
+pub fn login_with_password_as_parameter(
+    client: &mut TmcClient,
+    client_name: &str,
+    email: String,
+    password: String,
+) -> Result<Token, LangsError> {
+    let token = client.authenticate(client_name, email, password);
```

Comment:
More idiomatic to use Result::map_err like
```rust
    client
        .authenticate(client_name, email, password)
        .map_err(|e| LangsError::TmcClient(e))
```
than matching on the result when the Ok branch does nothing.